yt_dlp/extractor/fitnessblender.py

```diff
@@ -0,0 +1,25 @@
+from .brightcove import BrightcoveNewIE
+from .common import InfoExtractor
+
+
+class FitnessBlenderIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?fitnessblender\.com/videos/[\w-]+/(?P<tc>T)?(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://www.fitnessblender.com/page/fb-plus-player-test',

```

Comment:
Test doesn't match??

Reply:
The test still doesn't match your `_VALID_URL` regex